server/src/main/resources/application.yml

```diff
@@ -5,3 +5,7 @@ spring:
       host: "http://localhost/"
       port: 27017
       uri: "mongodb://localhost/paracord"
+
+schedule:
+  strandAvailabilityMS: 1000
```

Comment:
Is this 10 minutes? What unit of time does the scheduler use?

Reply:
Should have been milliseconds... makes your comment about comments really come to light lol

Reply:
And also 600000 should be the value